client/my-sites/plugins/main.jsx

```diff
@@ -422,16 +427,23 @@ export class PluginsMain extends Component {
 			if ( 'updates' === filterItem.id ) {
 				attr.count = this.props.pluginUpdateCount;
 			}
+			if ( 'all' === filterItem.id ) {
+				attr.count = this.props.currentPlugins.length;
+			}
 			return <NavItem { ...attr }>{ filterItem.title }</NavItem>;
 		} );
 
-		return (
-			<Main wideLayout>
-				<DocumentHead title={ this.props.translate( 'Plugins', { textOnly: true } ) } />
+		const pageTitle = this.props.translate( 'Plugins', { textOnly: true } );
+
+		const { isJetpackCloud } = this.props;
```

Comment:
Do we need this assignment here, as we have one at the beginning of the `renderPluginsContent` function?

Reply:
This is the `render` method, not `renderPluginsContent`